lapis/job_io/htcondor.py

```diff
@@ -17,20 +17,18 @@ def htcondor_job_reader(
     used_resource_name_mapping={  # noqa: B006
         "queuetime": "QDate",
         "walltime": "RemoteWallClockTime",  # s
-        "cores": "Number of Allocated Processors",
         "memory": "MemoryUsage",  # MB
         "disk": "DiskUsage_RAW",  # KiB
     },
     unit_conversion_mapping={  # noqa: B006
         "RequestCpus": 1,
         "RequestWalltime": 1,
-        "RequestMemory": 1.024 / 1024,
-        "RequestDisk": 1.024 / 1024 / 1024,
+        "RequestMemory": 1024 * 1024,
+        "RequestDisk": 1024,
         "queuetime": 1,
         "RemoteWallClockTime": 1,
-        "Number of Allocated Processors": 1,
-        "MemoryUsage": 1 / 1024,
-        "DiskUsage_RAW": 1.024 / 1024 / 1024,
+        "MemoryUsage": 1 / 1.048576 * 1024 * 1024,
```

Comment:
This needlessly computes an MiB value only to convert it to MB. You can do so directly:

```suggestion
        "MemoryUsage": 1000 * 1000,
```